### PR TITLE
update some dependencies.

### PR DESCRIPTION
**Pull request by ymo-sci**
httpclient is now included in the newer elasticsearch-rest-client in the until now provided version so we can remove it. setMaxWaitMillis is deprecated so we use setMaxWait now (which is internally the same)

- fixes: SIRI-920

### CODE REVIEW

File: src/main/java/sirius/db/jdbc/Database.java [hunk 1]
```diff
@@ -353,7 +353,7 @@ private void initialize() {
             ds.setMaxIdle(maxIdle);
             ds.setTestOnBorrow(testOnBorrow);
             ds.setValidationQuery(validationQuery);
-            ds.setMaxWaitMillis(1000);
+            ds.setMaxWait(Duration.ofMillis(1000));
```
Comment (sabieber) [inline src/main/java/sirius/db/jdbc/Database.java:356]: **Optional:** Why not `Duration.ofSeconds(1)`?

File: src/test/kotlin/sirius/db/testutil/MongoMocks.kt [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 package sirius.db.testutil
 
-import io.mockk.spyk
+import org.mockito.Mockito.spy
```
Comment (sabieber) [inline src/test/kotlin/sirius/db/testutil/MongoMocks.kt:11]: **Question:** But we still want to be able to use MockK?? Why is this change necessary?

**Review by mkeckmkeck — APPROVED** [review]

**Review by sabieber — APPROVED** [review]

**Review by jmuscireum — APPROVED** [review]
